python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -63,6 +63,8 @@ def from_pretrained(cls,
                                 or sym_int8. sym_int4 means symmetric int 4, asym_int4 means
                                 asymmetric int 4, etc. Relevant low bit optimizations will
                                 be applied to the model.
+        :param load_in_fp16: boolean value, True means load linear's weight to transposed format to
+                             take advantage of esimd optimizations of float16. Default to be False.
```

Comment:
Instead of adding a new `load_in_fp16` argument, just use `load_in_low_bit='fp8'`

Reply:
not sure why is `fp8` here, maybe we use `fp16` ?

Reply:
yes, fp16

Reply:
Done, have changed to `fp16`.